hr-time/idlharness.html

```diff
@@ -13,63 +13,35 @@
 <body>
 <h1>High Resolution Time IDL tests</h1>
 <div id="log"></div>
-
-<pre id='untested_idl' style='display:none'>
-[Exposed=Windows,Worker]
-interface WindowOrWorkerGlobalScope {
-};
-
-[Exposed=(Window,Worker)]
-interface EventTarget {
-};
-
-[Exposed=Window]
-interface PerformanceTiming {
-};
-
-[Exposed=Window]
-interface PerformanceNavigation {
-};
-
-// from nav-timing
-partial interface Performance {
-    [SameObject]
-    readonly attribute PerformanceTiming timing;
-    [SameObject]
-    readonly attribute PerformanceNavigation navigation;
-    [Default] object              toJSON();
-};
-</pre>
-
-<pre id='idl'>
-typedef double DOMHighResTimeStamp;
-
-[Exposed=(Window,Worker)]
-interface Performance : EventTarget {
-    DOMHighResTimeStamp now();
-    readonly attribute DOMHighResTimeStamp timeOrigin;
-    [Default] object              toJSON();
-};
-
-partial interface WindowOrWorkerGlobalScope {
-    [Replaceable]
-    readonly attribute Performance performance;
-};
-</pre>
-
 <script>
+'use strict';
 
-(function() {
+function doTest([dom, html, hr_time]) {
```

Comment:
`dom` is unused (you use dummy interfaces instead)

Reply:
Oops, will fix.